examples/transactions/examples/transfer_sign.rs

```diff
@@ -0,0 +1,42 @@
+use alloy::network::{eip2718::Encodable2718, TransactionBuilder};
+use alloy::primitives::utils::parse_units;
+use alloy::providers::Provider;
+use alloy::{
+    network::EthereumSigner, node_bindings::Anvil, primitives::U256, providers::ProviderBuilder,
+    rpc::types::eth::TransactionRequest, signers::wallet::LocalWallet,
+};
```

Comment:
Please use the `alloy::{...}` prelude crate only

If exports are missing from the `alloy` please flag them

Reply:
Great comments! I found the linter failed, so I copied and pasted what was there.
Let me know if this was what you meant 💪